src/components/AssetCard/AssetCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useMemo } from 'react'
+import { Network, Rarity, Item } from '@dcl/schemas'
+import { WearablePreview } from '../WearablePreview'
+import { Mana } from '../Mana/Mana'
+import { Card } from '../Card/Card'
+import { Profile } from '../Profile/Profile'
+import { formatWeiToAssetCard, getCatalogCardInformation } from './utils'
+import './AssetCard.css'
+
+export type AssetCardTranslations = {
+  also_minting: string
+  available_for_mint: string
+  available_listings_in_range: string
+  cheapest_listing: string
+  cheapest_option_range: string
+  cheapest_option: string
+  listing: string
+  listings: string
+  most_expensive_range: string
+  most_expensive: string
+  not_for_sale: string
+  owners: string
+  owner: string
+}
+
+export type AssetCardFilters = {
+  price?: number
+  sortBy?: string
+  minPrice?: string
+  maxPrice?: string
+}
+
+export type AssetCardProps = {
+  translations: AssetCardTranslations
```

Comment:
In other components, the translations are named as `i18n` and the types of the translations are nodes, would you mind changing the them to keep the consistency? Check [here](https://github.com/decentraland/ui/blob/master/src/components/BuyManaWithFiatModal/BuyManaWithFiatModal.tsx).
```suggestion
  i18n: AssetCardTranslations
```